project/blueprints/articles_blueprint.py

```diff
@@ -26,6 +26,7 @@ def details(article_id):
     return render_template('articles/details.html',
                            id=article_id,
                            title=article.title,
+                           subtitle=article.subtitle,
```

Comment:
```suggestion
                           article_subtitle=article.subtitle,
```
Take a look to project/templates/articles/form.html (line: 22) you need a value whose name is article_subtitle

Reply:
TY 😄